@here/harp-mapview/test/BoundsGeneratorTest.ts

```diff
@@ -91,7 +91,7 @@ describe("BoundsGenerator", function() {
 
     afterEach(async function() {
         if (mapView !== undefined) {
-            await mapView.getTheme(); // Needed otherwise the dispose will cause log messages
+            await mapView.loadPromise(); // Needed otherwise the dispose will cause log messages
```

Comment:
This looks like we should better fix the dispose method to dispose the promises correctly then to add a function for waiting for them. 

And expect a user to keep this order of API calls

Reply:
Yes, that is another good point. At least with the `loadPromise` approach, you can be sure that the `MapView` is in some valid usage state. 

By `await` ing the dispose, you are making the test easier, but users can't know when the `MapView` is fully loaded.

I see pro/contra for both sides.